webpack/assets/javascripts/react_app/redux/API/__tests__/__snapshots__/APISelectors.test.js.snap

```diff
@@ -1,5 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`API selectors should return the API current timestamp 1`] = `10000`;
```

Comment:
**Off topic, no need to act on it here :)**
I don't quite like using snapshots for simple values, just because it's convenient. That is systematic problem with our testing IMHO, we should think about this in the future. We are making the snapshot testing so easy, we make ourself think it's the only (or at least the best) way of testing.